srtcore/srt.h

```diff
@@ -99,7 +99,7 @@ written by
 extern "C" {
 #endif
 
-typedef int SRTSOCKET; // SRTSOCKET is a typedef to int anyway, and it's not even in UDT namespace :)
+typedef int32_t SRTSOCKET;
```

Comment:
This theoretically doesn't change anything, but it is better if it's specified explicitly with fixed size because since group are inserted, the single bits in the identifier will be used on purpse.